src/raylib.h

```diff
@@ -1370,6 +1370,7 @@ RLAPI void DrawSphereEx(Vector3 centerPos, float radius, int rings, int slices,
 RLAPI void DrawSphereWires(Vector3 centerPos, float radius, int rings, int slices, Color color);         // Draw sphere wires
 RLAPI void DrawCylinder(Vector3 position, float radiusTop, float radiusBottom, float height, int slices, Color color); // Draw a cylinder/cone
 RLAPI void DrawCylinderWires(Vector3 position, float radiusTop, float radiusBottom, float height, int slices, Color color); // Draw a cylinder/cone wires
+RLAPI void DrawArc(Vector3 v1, Vector3 v2, float iradius, float theta, int steps, Color color);          // Draws an arc between two points
```

Comment:
Also de function name could be confusing between 2D and 3D, maybe it could be renamed to `DrawArc3D()` or `DrawLineBezier3D()` to be aligned with the 2D implementation... but I understand it's not the same...

Reply:
Sure I can do that.  I realize now that the other 3D functions above it don't have 3D in the name because a 3D version of them is not possible, whereas an arc can be 2D or 3D.  I didn't do a 2D version precisely because you had 2D curves already.  But if you wanted a 2D version that would be easy to code up.

The thing that made the 3D version lengthy is that `Vector3Perpendicular()` doesn't suit for all orientations of the line between the two points, because of the way that `cardinalAxis` is chosen in that function, and the perpendicular line to a line in 3D is one of infintely many on the perpendicular plane.  For the 2D case, it's just one cross product and done.

Reply:
Probably just a `DrawArc3D()` would be enough.